cli/js/web/timers.ts

```diff
@@ -31,8 +31,15 @@ function clearGlobalTimeout(): void {
 
 let pendingEvents = 0;
 const pendingFireTimers: Timer[] = [];
-let hasPendingFireTimers = false;
-let pendingScheduleTimers: Timer[] = [];
+
+// Handler when timer macrotask is should be called
```

Comment:
JSDoc?  Even though this is internal, JSDoc does flow through better in editors that support it.